From 47f41dd57cee5ef2067e34b8be611b323bf2de9b Mon Sep 17 00:00:00 2001
From: David Reyna <David.Reyna@windriver.com>
Date: Thu, 21 Nov 2024 16:46:41 -0800
Subject: [PATCH] skip broken cbl-mariner database

Signed-off-by: David Reyna <David.Reyna@windriver.com>
---
 pkg/vulndb/db.go | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/pkg/vulndb/db.go b/pkg/vulndb/db.go
index 30a8d56..fb01328 100644
--- a/pkg/vulndb/db.go
+++ b/pkg/vulndb/db.go
@@ -77,6 +77,10 @@ func (t TrivyDB) Insert(targets []string) error {
 		}
 		log.Printf("Updating %s data...\n", target)
 
+		if ("cbl-mariner" == target) || ("azure" == target) {
+			continue
+		}
+
 		if err := src.Update(t.cacheDir); err != nil {
 			return xerrors.Errorf("%s update error: %w", target, err)
 		}
-- 
2.47.0
